include/stout/flags/flags.cc

```diff
@@ -587,18 +569,39 @@ void Parser::Parse(const std::vector<ArgumentInfo>& values) {
   }
 
   // Ensure required flags are present.
-  for (const auto& [_, field] : fields_) {
+  for (const auto& [name, field] : fields_) {
+    // The field has been parsed, all good, just continue.
+    if (parsed_.count(field) > 0) {
+      continue;
+    }
     const stout::v1::Flag& flag =
         field->options().GetExtension(stout::v1::flag);
 
-    // This field is not required, so just continue.
-    if (!flag.required())
+    // Need to normalize string default value before parsing.
+    // NOTE: for some reason protoc compiler has generated
```

Comment:
Maybe move this NOTE up to line 138 where you first reference `default_`, or maybe even better would be to duplicate this NOTE in both places where it's used (or better still might be to move this comment into a helper so there's only one line that accesses this weird field name).